css/ol.css

```diff
@@ -1,3 +1,8 @@
+.ol-box {
+  box-sizing: border-box;
+  border-radius: 2px;
+  border: 2px solid rgba(0,0,255,1);
```

Comment:
Because the old style had `color: [0, 0, 255, 1]`. But you're right, `blue` would be simpler.